Pipfile

```diff
@@ -9,6 +9,7 @@ verify_ssl = true
 python-gnupg = "*"
 click = "*"
 requests = "*"
+gpg = "*"
```

Comment:
The python gpg module is named `python-gnupg` and above. `gpg` is individual software.
Is there any reason for adding `gpg` to Pipfile? Also, I tried and found this will cause build to fail.